Formula/mesa.rb

```diff
@@ -11,44 +11,45 @@ class Mesa < Formula
 
   option "without-test", "Skip compile-time tests"
   option "with-static", "Build static libraries (not recommended)"
+  option "with-llvm", "Build drivers that require llvm and libelf"
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
   depends_on "python" => :build
```

Comment:
Please change this to `depends_on "python" => :build`

The `python` formula (python 3) is currently broken.

We can have a look later if it makes sense to build this with python3.

Reply:
I suspect that Michka meant `python@2` here.

Reply:
I think so too :)

Reply:
Yes, sorry, `python@2` :)